### DISCUSSION TITLE
Plugin installed but not listed among filters?

### DISCUSSION THREAD

**Discussion by jocelynetienne** [royshil/obs-backgroundremoval#604] [Q&A]
I have `obs` 29.0.2 and `obs-backgroundremoval` both installed via debian `apt`

```
i   obs-backgroundremoval                            - obs-backgroundremoval built using CMake 
i   obs-studio                                       - recorder and streamer for live video content               
```

It seems it has worked:
```
info: [obs-backgroundremoval] Plugin loaded successfully (version 1.1.13)
info:     obs-backgroundremoval.so
```

However, when looking for filters for my cam, Background Removal is not listed:
![image](https://github.com/user-attachments/assets/b6bdcdb4-91e4-404f-8e06-42c6417bd2c5)

**Comment by royshil** [comment]
OBS v30+ is required
can you share logs?
